osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPoint.cs

```diff
@@ -50,6 +50,7 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            Delay(-TransformStartTime); // So our follow points start their transforms from t=0 (the start of the map), rather from the lead in time (where the clock is right now).

```

Comment:
Isn't the problem the other way round right now? Follow points are currently working (in master) because it assumes that the clock is set to 0 on load, which happens to be in the case in master. This makes sure that we start our transforms on 0.

`TransformStartTime` is not set, it's a property that reads from the clock and the transformDelay, which are both local to the object aren't they?

```
        private double transformDelay;
...
        protected double TransformStartTime => Clock != null ? Time.Current + transformDelay : 0;
```